Doc/library/json.rst

```diff
@@ -175,6 +175,8 @@ Basic Usage
    .. versionchanged:: 3.4
       Use ``(',', ': ')`` as default if *indent* is not ``None``.
 
+   .. versionadded:: 3.7 Instead of ``(',', ':')`` constant ``json.COMPACT`` could be used.
```

Comment:
Also, following the style used in `Doc/library/json.rst` would be better:

```rst
.. versionadded:: 3.7
   Instead of ``(',', ':')`` constant ``json.COMPACT`` could be used.
```